### PR TITLE
fix delete 2 repos in a row

### PR DESCRIPTION
**Pull request by juliangruber**
Two parts were required to this:

- create a new `ConfirmModal` instance each time, because otherwise old instances would be recycled inproperly. It's not an issue perf wise to create a modal class every time for us fortunately
- use `send()` to set the uri back instead of `window.location.back`, which i think caused a race condition

### CODE REVIEW

**Comment by juliangruber** [comment]
not fixed

**Comment by juliangruber** [comment]
seems fixed!

**Comment by juliangruber** [comment]
I fixed this by bypassing the routing layer for modals completely - which in my understanding isn't necessary anyway

File: elements/link-modal.js [hunk 1]
```diff
@@ -120,12 +120,18 @@ function createModal () {
...
+  const modal = Modal({
+    onunload,
+    onexit,
+    render,
+    class: 'modal'
+  })
 
   return modal
 
   function onexit () {
```
Comment (juliangruber) [inline elements/link-modal.js]: can be refactored if we get a reference to the current element here, either via context or an argument:

```js
function onexit () {
  this.parentNode.removeChild(this)
}
```

or

```js
function onexit (el) {
  el.parentNode.removeChild(el)
}
```

I'd prefer the former

Comment (karissa) [inline elements/link-modal.js]: yeah the former makes sense

**Comment by yoshuawuyts** [comment]
I've got some concerns by making UI elements in models, but happy to merge for now and I'll refactor it in a subsequent patch. Let's get this puppy out there :sparkles:

File: elements/link-modal.js (hunk 1 above)
Comment (yoshuawuyts) [inline elements/link-modal.js]: Not a fan of using `this` in any form external facing API; feel it's best left for internals

**Pull request merged**